### PR TITLE
Add missing Blockchain Node APIs to ain-js

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Split getProposer() to getProposerByNumber() and getProposerByHash()
- Split getValidators() to getValidatorsByNumber() and getValidatorsByHash()
- Add getValidatorInfo() method

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1248

### CODE REVIEW

File: src/ain.ts [hunk 1]
```diff
@@ -165,27 +165,48 @@ export default class Ain {
...
+   * Fetches the validator list of a block with a block hash.
+   * @param {string} blockHash The block hash.
+   * @returns {Promise<any>}
+   */
+  getValidatorsByHash(blockHash: string): Promise<any> {
+    return this.provider.send('ain_getValidatorsByHash', { hash: blockHash });
+  }
+
+  /**
+   * Fetches the forger's address of a block with a block number.
```
Comment (jiyoung-an) [inline src/ain.ts:195]: question) what is `forger`?

**Comment by jiyoung-an** [comment]
LGTM with some question!

**Review by jiyoung-an — APPROVED** [review]

File: src/ain.ts (hunk 1 above)
Comment (platfowner) [reply src/ain.ts]: sometimes block proposer is called block forger or block producer. just replaced with 'block proposer' for readability. thanks!

**Comment by platfowner** [comment]
Thanks for the review! @jiyoung-an Now merging..

**Pull request merged**